packages/edge/README.md

```diff
@@ -148,6 +148,15 @@ See [MobileCore.resetIdentities](../core/README.md#resetidentities) for more det
 
 ### sendEvent
 
+Sends an Experience event to Edge Network.
+
+Starting with **@adobe/react-native-aepedge v5.1.0**, the sendEvent API supports optional Datastream overrides. This allows you to adjust your datastreams without the need for new ones or modifications to existing settings. The process involves two steps:
+
+1. Define your Datastream configuration overrides on the [datastream configuration page](https://experienceleague.adobe.com/docs/experience-platform/datastreams/configure.html).
```

Comment:
Looks like the new link is this one: https://experienceleague.adobe.com/docs/experience-platform/datastreams/overrides.html